tests/unittests/test_k2.py

```diff
@@ -0,0 +1,369 @@
+import os
+import pytest
+import tempfile
+import shutil
+import torch
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+try:
+    import k2
+except ImportError:
+    raise ImportError("Please install k2 with `pip install k2`")
```

Comment:
I suggest that you put the following link here.
https://k2-fsa.github.io/k2/installation/from_wheels.html

k2 is compiled against different versions of CUDA toolkit and PyTorch.

PyPI supports only hosting a specific version of CUDA and PyTorch.
At most of the time, if not all of the time, `pip install k2` installs a version that is not compatible with your locally installed CUDA toolkit and PyTorch.

Reply:
Thanks @csukuangfj, @ZhaoZeyu1995 could you add this URL to the log?

Reply:
No problem! I just made a commit regarding it.